api/desecapi/serializers.py

```diff
@@ -245,12 +246,16 @@ def get_validators(self):
 
     @staticmethod
     def validate_type(value):
-        if value in models.RRset.DEAD_TYPES:
-            raise serializers.ValidationError(f'The {value} RRset type is currently unsupported.')
-        if value in models.RRset.RESTRICTED_TYPES:
-            raise serializers.ValidationError(f'You cannot tinker with the {value} RRset.')
-        if value.startswith('TYPE'):
-            raise serializers.ValidationError('Generic type format is not supported.')
+        if value not in models.RR_SET_TYPES_MANAGEABLE:
+            # TODO see if SMIMEA can become a manageable type
```

Comment:
move comment to `models.RR_SET_TYPES_MANAGEABLE` declaration? -- but there, it is listed as supported?

Reply:
deleted without replacement; if dnspython starts supporting SMIME so do we -- tests are already prepared